interfaces/session_cost_calculator.yaml

```diff
@@ -0,0 +1,11 @@
+description: >-
+  This interface publishes the running or finished session costs. This interface
```

Comment:
I think the "calculator" is not necessary - it should be clear since its an interface

Reply:
Agree. Renames.